baldrick/scripts/stale_issues.py

```diff
@@ -59,9 +59,22 @@ def process_issues(repository, installation,
         if labeled_time is None:
             continue
 
-        dt = now - labeled_time
+        time_since_close_label = now - labeled_time
 
-        if dt > close_seconds:
+        # Note: if warning time is before label time, it's as if the warning
+        # didn't exist since it's no longer relevant.
+        warning_time = issue.last_comment_date(f'{bot_name}[bot]', filter_keep=is_close_warning)
+        if warning_time is None or warning_time < labeled_time:
+            time_since_last_warning = -1.
+        else:
+            # We use max() here to make sure that the value is positive
+            time_since_last_warning = max(0, now - warning_time)
+
+        # We only close pull requests if there has been a warning before, and
+        # the time since the warning exceeds the threshold specified by
+        # stale_pull_requests_close_seconds.
```

Comment:
update comment for issues rather than pull requests

Reply:
Fixed!